static/app/components/events/interfaces/crashContent/stackTrace/content.tsx

```diff
@@ -206,6 +291,24 @@ class Content extends Component<Props, State> {
       platform === 'java' && event.tags?.find(({key}) => key === 'mechanism')?.value;
     const isANR = mechanism === 'ANR' || mechanism === 'AppExitInfo';
 
+    let keys: number[] = [];
+    Object.keys(toggleFrameMap)
```

Comment:
This code block is a bit difficult to parse. Can you extract it to its own function and give it a descriptive name?